misc/scripts/deploy-tools.sh

```diff
@@ -210,6 +211,8 @@ function deployESXInstall {
 }
 
 function deployESXPost {
+    # restart hostd so that esxcli extension file is read
+    $SSH $TARGET $HOSTD restart
```

Comment:
As discussed offline .. it should be part of installESX routine; need to move there.